spec/active_record/monetizable_spec.rb

```diff
@@ -12,7 +12,9 @@ class Sub < Product; end
                      sale_price_amount: 1200, delivery_fee_cents: 100,
                      restock_fee_cents: 2000,
                      reduced_price_cents: 1500, reduced_price_currency: :lvl,
-                     lambda_price_cents: 4000)
+                     lambda_price_cents: 4000,
+                     unit_cost_cents: 1234.8765
```

Comment:
It would be good to pass a fractional value to an existing column that is not stored as a decimal (like `price_cents`), to demonstrate that it will still round values as it used to.

```
price_cents: 3000.14
```

Reply:
Actually, `price_cents` is the direct accessor for the column, so it will truncate.  But, using the accessor created by `monetize` should round, so I will add `price: 30.0051` which should get rounded to `3001` in cents.

Reply:
Incorporated above into the new precision setting tests.